integration-tests/src/test/java/org/jboss/pnc/bacon/test/AbstractTest.java

```diff
@@ -34,27 +34,27 @@
     protected PNCWiremockHelper wmock = new PNCWiremockHelper();
 
     @BeforeAll
-    public static void startWiremockServer() {
+    static void startWiremockServer() {
         wireMockServer = new WireMockServer(options().port(8080).notifier(new ConsoleNotifier(false)));
         wireMockServer.start();
     }
 
     @AfterAll
-    public static void stopWiremockServer() {
+    static void stopWiremockServer() {
         wireMockServer.stop();
     }
 
     @BeforeEach
-    public void stubBabnner() {
+    void stubBanner() {
         wmock.init();
     }
 
     @AfterAll
-    public void clearStubs() {
+    static void clearStubs() {
```

Comment:
Is one of these supposed to be `@AfterEach` (there are two `@AfterAll` here)?

Reply:
Looks like it should be AfterEach right? It does feel like that would be the right one.